src/fs/dir.rs

```diff
@@ -129,11 +129,24 @@ impl<'dir, 'ig> Files<'dir, 'ig> {
                     }
                 }
 
-                return Some(File::from_args(path.clone(), self.dir, filename, self.deref_links)
-                                 .map_err(|e| (path.clone(), e)))
+                let file = File::from_args(
+                    path.clone(),
+                    self.dir,
+                    filename,
+                    self.deref_links
+                ).map_err(|e| (path.clone(), e));
+
+                // Windows has it's own concept of hidden files, with a flag
```

Comment:
Nit: should be "its", no apostrophe

Reply:
ugh I knew that as well... all sorted though